code/modules/food_and_drinks/food/foods/sandwiches.dm

```diff
@@ -136,18 +143,171 @@
 	list_reagents = list("nutriment" = 40, "vitamin" = 5)
 	tastes = list("bun" = 4, "meat" = 2, "cheese" = 2, "type two diabetes" = 10)
 
-/obj/item/reagent_containers/food/snacks/jellyburger
+/obj/item/reagent_containers/food/snacks/burger/crazy
+	name = "crazy hamburger"
+	desc = "This looks like the sort of food that a demented clown in a trenchcoat would make."
+	icon_state = "crazyburger"
+	filling_color = "#E3D681"
+	bitesize = 3
+	list_reagents = list("nutriment" = 10, "vitamin" = 2, "capsaicin" = 3, "condensedcapsaicin" = 2)
+	tastes = list("bun" = 2, "meat" = 4, "cheese" = 2, "beef soaked in chili" = 3, "a smoking flare" = 2)
+
+/obj/item/reagent_containers/food/snacks/burger/ppattywhite
+	name = "white pretty patty"
+	desc = "Delicious titanium!"
+	icon_state = "ppatty-mime"
+	filling_color = "#FFFFFF"
+	bitesize = 2
+	list_reagents = list("nutriment" = 7, "protein" = 1)
+	tastes = list("bun" = 2, "meat" = 2, "white" = 2)
+
+/obj/item/reagent_containers/food/snacks/burger/ppattyred
+	name = "red pretty patty"
+	desc = "Perfect for hiding the fact that it's burnt to a crisp."
+	icon_state = "ppatty-red"
+	filling_color = "#D63C3C"
+	bitesize = 2
+	list_reagents = list("nutriment" = 7, "protein" = 1)
+	tastes = list("bun" = 2, "meat" = 2, "red" = 2)
+
+/obj/item/reagent_containers/food/snacks/burger/ppattyorange
+	name = "orange pretty patty"
+	desc = "Contains 0% juice."
+	icon_state = "ppatty-orange"
+	filling_color = "#FFA500"
+	bitesize = 2
+	list_reagents = list("nutriment" = 7, "protein" = 1)
+	tastes = list("bun" = 2, "meat" = 2, "orange" = 2)
+
+/obj/item/reagent_containers/food/snacks/burger/ppattyyellow
+	name = "yellow pretty patty"
+	desc = "Bright to the last bite."
+	icon_state = "ppatty-yellow"
+	filling_color = "#FFFF00"
+	bitesize = 2
+	list_reagents = list("nutriment" = 7, "protein" = 1)
+	tastes = list("bun" = 2, "meat" = 2, "yellow" = 2)
+
+/obj/item/reagent_containers/food/snacks/burger/ppattygreen
+	name = "green pretty patty"
+	desc = "It's not tainted meat, it's painted meat!"
+	icon_state = "ppatty-green"
+	filling_color = "#00FF00"
+	bitesize = 2
+	list_reagents = list("nutriment" = 7, "protein" = 1)
+	tastes = list("bun" = 2, "meat" = 2, "green" = 2)
+
+/obj/item/reagent_containers/food/snacks/burger/ppattyblue
+	name = "blue pretty patty"
+	desc = "Is this blue rare?"
+	icon_state = "ppatty-blue"
+	filling_color = "#0000FF"
+	bitesize = 2
+	list_reagents = list("nutriment" = 7, "protein" = 1)
+	tastes = list("bun" = 2, "meat" = 2, "blue" = 2)
+
+/obj/item/reagent_containers/food/snacks/burger/ppattypurple
+	name = "purple pretty patty"
+	desc = "Regal and low class at the same time."
+	icon_state = "ppatty-purple"
+	filling_color = "#800080"
+	bitesize = 2
+	list_reagents = list("nutriment" = 7, "protein" = 1)
+	tastes = list("bun" = 2, "meat" = 2, "purple" = 2)
+
+/obj/item/reagent_containers/food/snacks/burger/ppattyrainbow
```

Comment:
Not that I disagree, but I'm not quite sure what they would inherit here?

Reply:
It just makes it better organized imo. We have a lot of shit thats similar, but dont share a path because of old coders. It also makes it easier for getting all of them, as like typesof() or subtypes() of

Reply:
Ahh, alright, fair point and reasonable. Totally don't mind doing it at all!